controller_handler.py

```
@@ -0,0 +1,31 @@
+from typing import Optional
+import discord
+from discord.ext import commands
```

Flake8 (7.2.0)

[error] 'discord.ext.commands' imported but unused

(F401)